docs/data/material/getting-started/templates/sign-up/CustomIcons.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react';
+import { SxProps, Theme } from '@mui/system';
+import SvgIcon from '@mui/material/SvgIcon';
+
+interface IconProps {
+  sx?: SxProps<Theme>;
+}
+
+export function SitemarkIcon({ sx }: IconProps) {
+  /* this logo is from LogoToUse.com */
+  return (
+    <SvgIcon sx={sx}>
```

Comment:
```suggestion
    <SvgIcon>
```

Why do we need the `sx` prop llfted up here? We don't seem to be using it:

``` 
startIcon={<GoogleIcon />}
```

Reply:
I'd also say it's fine to remove the commented note there — not sure we need to link to the source we got the SVGs from. It'd be best if we got them from "more official" sources, too, though, like from a branding website from Google and Facebook themselves. I was about to comment that the FB svg looks a bit weird for some reason.

Reply:
I updated Google and Facebook logos 👍 The comment with source was only about Sitemark, though.